pkg/cstor/cstor.go

```diff
@@ -526,6 +537,12 @@ func (p *Plugin) CreateVolumeFromSnapshot(snapshotID, volumeType, volumeAZ strin
 	}
 
 	if newVol.restoreStatus == v1alpha1.RSTCStorStatusDone {
+		if p.autoSetTargetIP {
+			if err := p.markCVRsAsRestoreCompleted(newVol); err != nil {
+				return newVol.volname, err
```

Comment:
Better to wrap error with message, like "Error setting targetip on CVR, need to set it manually Refer:<README LINK?>".

Reply:
done, PTAL